examples/typescript-ex/src/index.ts

```diff
@@ -1,45 +1,48 @@
-import { CloudEvent, HTTPREceiver } from '../../';
+import { CloudEvent, HTTPReceiver } from '../../..';
+import { CloudEventV1 } from '../../../lib/v1';
 
 export function doSomeStuff() {
-  const receiver = new HTTPREceiver();
+  const receiver = new HTTPReceiver();
 
-  const myevent: CloudEvent = new CloudEvent()
-    .source('/source')
-    .type('type')
-    .dataContentType('text/plain')
-    .dataschema('http://d.schema.com/my.json')
-    .subject('cha.json')
-    .data('my-data')
-    .addExtension("my-ext", "0x600");
+  const myevent: CloudEventV1 = new CloudEvent({
+    source: '/source',
+    type: 'type',
+    dataContentType: 'text/plain',
+    dataSchema: 'https://d.schema.com/my.json',
+    subject: 'cha.json',
+    data: 'my-data'
+  });
+  myevent.addExtension('extension-1', 'some extension data');
```

Comment:
I prefer single quotes, but the project and the rest of this example is double quotes. Maybe these should be changed here. (I know it was already like that, but...).

Reply:
yea,  there is a "prettier" config in this example that wants you to use single quotes.  I'll probably just remove that and update to double quotes to keep things consistent